config/samples/network_v1alpha2_bgpconf.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: network.kubesphere.io/v1alpha2
+kind: BgpConf
+metadata:
+  #The porter only recognizes configurations with default names;
+  #configurations with other names are ignored.
+  name: default
```

Comment:
How about set the name to `porter`

Reply:
This is fine, but the name `default` is used both in the code and in the documentation, so I don't want to change it, and it's not really perceptible to the user when it's configured through the UI.